src/admin/index.js

```diff
@@ -426,6 +429,85 @@ module.exports = ({
       .pop()
   }
 
+  /**
+   * @param {string} groupId
+   * @param {string[]} topics
+   * @param {boolean} [resolveOffsets=false]
+   * @return {Promise}
+   */
+  const fetchConsumerGroupOffsets = async ({ groupId, topics = [], resolveOffsets = false }) => {
+    if (!groupId) {
+      throw new KafkaJSNonRetriableError(`Invalid groupId ${groupId}`)
+    }
+
+    if (!Array.isArray(topics)) {
```

Comment:
In the previous implementation if `topic` wasn't passed an exception was thrown
```
    if (!topic) {
      throw new KafkaJSNonRetriableError(`Invalid topic ${topic}`)
    }
```
but here you're setting `topics`'s default value to pass your exception throwing (`[]`), so you're actually not validating that this param was sent here

Reply:
mm yeah but I think it's kinda weird to force users to set topics=[] if they want to just fetch all of topics of the consumer instead of just calling it with `fetchConsumerGroupOffsets({groupId})`.